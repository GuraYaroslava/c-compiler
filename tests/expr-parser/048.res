tree 1:
funtion: +
args: 
    a
+ <
    b


