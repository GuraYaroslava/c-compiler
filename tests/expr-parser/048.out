tree 1:
funtion: 
                   a
         + <
                   b
args: 
                   a
         + <
                   b


